back/src/main/java/com/baba/back/content/service/ContentService.java

```diff
@@ -0,0 +1,84 @@
+import com.baba.back.content.exception.ContentAuthorizationException;
+import com.baba.back.content.repository.ContentRepository;
+import com.baba.back.oauth.domain.member.Member;
+import com.baba.back.oauth.exception.MemberNotFoundException;
+import com.baba.back.oauth.repository.MemberRepository;
+import com.baba.back.relation.domain.Relation;
+import com.baba.back.relation.exception.RelationNotFoundException;
+import com.baba.back.relation.repository.RelationRepository;
+import java.time.LocalDate;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class ContentService {
+
+    private final ContentRepository contentRepository;
+    private final MemberRepository memberRepository;
+    private final BabyRepository babyRepository;
+    private final RelationRepository relationRepository;
+    private final ImageSaver imageSaver;
+
+    public CreateContentResponse createContent(CreateContentRequest request, String memberId, String babyId) {
+        final Member member = validateMember(memberId);
+
+        final Baby baby = validateBaby(babyId);
+
+        final Relation relation = validateRelation(memberId, babyId, member, baby);
+
+        checkAuthorization(memberId, babyId, relation);
+
+        final Content content = Content.builder()
+                .title(request.getTitle())
+                .contentDate(request.getDate())
+                .now(LocalDate.now())
+                .cardStyle(request.getCardStyle())
+                .imageSource("")
+                .baby(baby)
+                .build();
+
+        final ImageFile imageFile = new ImageFile(request.getPhoto());
+        final String imageSource = imageSaver.save(imageFile);
+
+        content.updateURL(imageSource);
+
+        contentRepository.save(content);
+
+        return new CreateContentResponse(true);
+    }
+
+    private Member validateMember(String memberId) {
+        return memberRepository.findById(memberId)
+                .orElseThrow(() -> new MemberNotFoundException(memberId + "에 해당하는 멤버가 존재하지 않습니다."));
+    }
+
+    private Baby validateBaby(String babyId) {
+        return babyRepository.findById(babyId)
+                .orElseThrow(() -> new BabyNotFoundException(babyId + " 는 존재하지 않는 babyId 입니다."));
+    }
+
+    private Relation validateRelation(String memberId, String babyId, Member member, Baby baby) {
+        return relationRepository.findByMemberAndBaby(member, baby)
+                .orElseThrow(() -> new RelationNotFoundException(
+                        memberId + "와 " + babyId + " 사이의 관계가 존재하지 않습니다."));
+    }
+
+    private static void checkAuthorization(String memberId, String babyId, Relation relation) {
+        if (!relation.isFamily()) {
+            throw new ContentAuthorizationException(memberId + "는 " + babyId + "와 가족 관계가 아닙니다.");
```

Comment:
```suggestion
    private static void checkAuthorization(Relation relation) {
        if (!relation.isFamily()) {
            throw new ContentAuthorizationException(relation.getId() + " 관계는 가족 관계가 아닙니다.");
```

이렇게만 해도 충분히 로그를 해석할 수 있지 않을까요?

Reply:
충분한 것 같습니다!